src/index.js

```diff
@@ -187,6 +187,8 @@ const userEvent = {
 
     const previousText = element.value;
 
+    element.focus();
```

Comment:
Rather than focusing I think we should verify that the `element === document.activeElement` and throw if not as mentioned here: https://github.com/testing-library/user-event/issues/178#issuecomment-608324794

Reply:
If we remove `element.focus()`, then how could we call `onFocus` callback will?